R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
Sorry for being slow, I had never thought about a build time stuff like that. Of course the code is executed during the install. 
Should we keep we keep the imported color functions as Gabor suggested ?